components/clp-package-utils/clp_package_utils/controller.py

```
@@ -522,6 +524,35 @@ def _set_up_env_for_webui(self, container_clp_config: CLPConfig) -> EnvVarsDict:
 
         return env_vars
 
+    def _set_up_env_for_mcp_server(self) -> EnvVarsDict:
+        """
+        Sets up environment variables for the MCP server component.
+
+        :return: Dictionary of environment variables necessary to launch the component.
+        """
+        component_name = MCP_SERVER_COMPONENT_NAME
+        logger.info(f"Setting up environment for {component_name}...")
```

Ruff (0.14.1)

Logging statement uses f-string

(G004)